### ISSUE TITLE
volumes that don't get destroyed

### ISSUE THREAD

**Issue by psftw** [cpuguy83/docker-volumes#10]
I just purged hundreds of orphan volumes and four don't want to die.  

```Shell
$ docker-volumes list
       ID      | NAMES |                                           PATH
+--------------+-------+------------------------------------------------------------------------------------------+
  b2127ce5f2d7 |       | /var/lib/docker/vfs/dir/b2127ce5f2d7c80f47c3185e3dfb2df585d8c6d04399b63d5bacd1f5e7e8ddce
  bb82bb1ef5bd |       | /var/lib/docker/vfs/dir/bb82bb1ef5bd9f26ffa34f8492ce83a9f205062b90de9aa2aa9aa8002fd5c1c4
  dfae89dd21cb |       | /var/lib/docker/vfs/dir/dfae89dd21cb76afdc952956b4d27846e926932b5f93bf19c8735963962d536b
  3208b572e2ea |       | /var/lib/docker/vfs/dir/3208b572e2ea24e5c36f15ad4a037861f22e848421887e6612113438da394476
$ docker-volumes rm $(docker-volumes list -q)
Successfully removed volume:  3208b572e2ea24e5c36f15ad4a037861f22e848421887e6612113438da394476
Successfully removed volume:  b2127ce5f2d7c80f47c3185e3dfb2df585d8c6d04399b63d5bacd1f5e7e8ddce
Successfully removed volume:  bb82bb1ef5bd9f26ffa34f8492ce83a9f205062b90de9aa2aa9aa8002fd5c1c4
Successfully removed volume:  dfae89dd21cb76afdc952956b4d27846e926932b5f93bf19c8735963962d536b
$ docker-volumes list
       ID      | NAMES |                                           PATH
+--------------+-------+------------------------------------------------------------------------------------------+
  b2127ce5f2d7 |       | /var/lib/docker/vfs/dir/b2127ce5f2d7c80f47c3185e3dfb2df585d8c6d04399b63d5bacd1f5e7e8ddce
  bb82bb1ef5bd |       | /var/lib/docker/vfs/dir/bb82bb1ef5bd9f26ffa34f8492ce83a9f205062b90de9aa2aa9aa8002fd5c1c4
  dfae89dd21cb |       | /var/lib/docker/vfs/dir/dfae89dd21cb76afdc952956b4d27846e926932b5f93bf19c8735963962d536b
  3208b572e2ea |       | /var/lib/docker/vfs/dir/3208b572e2ea24e5c36f15ad4a037861f22e848421887e6612113438da394476
$ sudo ls -hl /var/lib/docker/vfs/dir/b2127ce5f2d7c80f47c3185e3dfb2df585d8c6d04399b63d5bacd1f5e7e8ddce
total 84K
drwx------ 5 999 docker 4.0K Sep 19 12:03 base
drwx------ 2 999 docker 4.0K Sep 19 12:03 global
drwx------ 2 999 docker 4.0K Sep 19 12:03 pg_clog
-rw------- 1 999 docker 4.5K Sep 19 12:03 pg_hba.conf
-rw------- 1 999 docker 1.6K Sep 19 12:03 pg_ident.conf
drwx------ 4 999 docker 4.0K Sep 19 12:03 pg_multixact
drwx------ 2 999 docker 4.0K Sep 19 12:03 pg_serial
drwx------ 2 999 docker 4.0K Sep 19 12:03 pg_snapshots
drwx------ 2 999 docker 4.0K Sep 19 16:55 pg_stat_tmp
drwx------ 2 999 docker 4.0K Sep 19 12:03 pg_subtrans
drwx------ 2 999 docker 4.0K Sep 19 12:03 pg_tblspc
-rw------- 1 999 docker    4 Sep 19 12:03 PG_VERSION
drwx------ 3 999 docker 4.0K Sep 19 12:03 pg_xlog
-rw------- 1 999 docker  21K Sep 19 12:03 postgresql.conf
-rw------- 1 999 docker   37 Sep 19 12:03 postmaster.opts
```

These are easy enough to kill off manually, but if you are interested I could try to help debug why docker-volumes is unable to delete them.

```Shell
$ docker info
Containers: 0
Images: 1604
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Dirs: 1604
Execution Driver: native-0.2
Kernel Version: 3.13.0-36-generic
Operating System: Ubuntu 14.04.1 LTS
WARNING: No swap limit support
$ docker version
Client version: 1.2.0
Client API version: 1.14
Go version (client): go1.3.1
Git commit (client): fa7b24f
OS/Arch (client): linux/amd64
Server version: 1.2.0
Server API version: 1.14
Go version (server): go1.3.1
Git commit (server): fa7b24f
```

**Comment by billsmith** [comment]
+1 Same problem here.  The most recent attempt took 18 tries before the volume was actually deleted.

**Comment by cpuguy83** [comment]
Can you test with #15 ?

**Comment by billsmith** [comment]
@cpuguy83 Before, almost every volume I tried to delete required multiple attempts.  With #15, I've deleted over 100 volumes, all on the first attempt.  Thank you.

I did not test the help, inspect, export, or import commands.

**Comment by cpuguy83** [comment]
Released v1.1.1, binaries are available in the releases section.

**Comment by thaJeztah** [comment]
:heart: Thanks a lot.
